client/landing/subscriptions/components/site-list/styles.scss

```diff
@@ -74,6 +74,19 @@ $max-list-width: 1300px;
 					}
 				}
 			}
+
+			.p2-label {
+				display: inline-block;
+				text-align: center;
+				width: 36px;
+				height: 20px;
+				background: #2a6aee;
+				color: $studio-green-0;
+				font-size: 12;
```

Comment:
The `font-size` value is missing `px`, but regardless, we can use variable instead. I think `$font-body-extra-small` should be the right one.

Reply:
Good point! Using the variable now.